### ISSUE TITLE
application crash on EVO

### ISSUE THREAD

**Issue by GoogleCodeExporter** [BinaryBlue/imsdroid#55]
```
What steps will reproduce the problem?
1.make video call
2.flip screen from vertical to horizontal or vice versa
3.application crashes with unexpected error

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?


Please provide any additional information below.


```

Original issue reported on code.google.com by `chris.ve...@gmail.com` on 10 Sep 2010 at 3:20

**Comment by GoogleCodeExporter** [comment]
```
Hi,
Could you please send the log file using "Log collector" application from the 
market?
My email address is diopmamadou(at)doubango(dot)org.
Thanks
```

Original comment by `boss...@yahoo.fr` on 12 Sep 2010 at 4:37
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.0.292.
```

Original comment by `boss...@yahoo.fr` on 28 Sep 2010 at 9:28
- Changed state: **Fixed**
